app/experimenter/experiments/constants.py

```diff
@@ -27,7 +27,7 @@ class ExperimentConstants(object):
     STATUS_DRAFT_LABEL = "Draft"
     STATUS_REVIEW_LABEL = "Ready for Sign-Off"
     STATUS_SHIP_LABEL = "Ready to Ship"
-    STATUS_ACCEPTED_LABEL = "Accepted by Normandy"
+    STATUS_ACCEPTED_LABEL = "Accepted by Shield"
```

Comment:
Wasn't it better with "Normandy"?

Reply:
The second commit changes it back: https://github.com/mozilla/experimenter/pull/1098/commits/e0906a0cf29f32579fe64119304456632c83e2f7#diff-00de22e6c754b3d410f1d3e631e1825eR30

Reply:
Oops. I just have clicked on the wrong one.